myuw/static/js/landing.js

```diff
@@ -72,6 +72,10 @@ var Landing = {
             TacomaApplicationCard
         ];
 
+        // if (window.user.intl_stud) {
```

Comment:
Should this (and the mobile one below) be commented out?

Reply:
No, that's for the next release.